stdlib/public/Concurrency/Task.swift

```diff
@@ -194,18 +194,29 @@ extension Task {
     }
   }
 
-  /// Indicates that the task should stop running.
+  /// Marks this task as cancelled.
```

Comment:
Documentation follows [Apple Style Guide](https://support.apple.com/guide/applestyleguide/c-apsgb744e4a3/1.0/web/1.0) which calls for spelling "canceled" with one L, not two.

```suggestion
  /// Marks this task as canceled.
```

The original abstract avoided repeating the word "cancel" because of feedback from editorial, that we don't want to just restate a symbol name in its abstract.  Am I correct in assuming that you're changing it because it's up to the developer what cancellation means?

Reply:
> Am I correct in assuming that you're changing it because it's up to the developer what cancellation means?

yes, we don't really "stop running", so it's up to developers to do something about the cancellation